scripts/lint.sh

```diff
@@ -0,0 +1,3 @@
+# a line length of 88 is black's default
+flake8 --max-line-length 88 selection
```

Comment:
90, 88 is bad.

Reply:
Do you know why they chose it as default? Why is 88 bad?

Reply:
Or 86.

Reply:
> Do you know why they chose it as default? Why is 88 bad?

Yes.
I would be happier using 86 oder the common 80, because we do not have the google problem with many indentation levels